### PR TITLE
[Docs] Deployment translation

### PR DESCRIPTION
**Pull request by xin-li-67**
## Motivation

Translate deployment docs

## Modification

modified:   docs/en/deploy/basic_deployment_guide.md
modified:   docs/en/deploy/yolov5_deployment.md
modified:   docs/zh_cn/deploy/basic_deployment_guide.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # YOLOv5 部署全流程说明
 
-请先参考 [`部署必备指南`](./部署必备指南.md) 了解部署配置文件等相关信息。
+请先参考 [`部署必备指南`](basic_deployment_guide.md) 了解部署配置文件等相关信息。
 
 ## 模型训练和测试
 
-模型训练和测试请参考 [`YOLOv5 从入门到部署全流程`](docs/zh_cn/user_guides/yolov5_tutorial.md) 。
+模型训练和测试请参考 [YOLOv5 从入门到部署全流程](../../zh_cn/user_guides/yolov5_tutorial.md) 。
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:7]:
```suggestion
模型训练和测试请参考 [YOLOv5 从入门到部署全流程](../user_guides/yolov5_tutorial.md) 。
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 当您需要部署静态输入模型时，您应该确保模型的输入尺寸是固定的，比如在测试流程或测试数据集加载时输入尺寸为 `640x640`。
 
-您可以查看 [`yolov5_s-deploy.py`](configs/deploy/model/yolov5_s-deploy.py) 中测试流程或测试数据集加载部分，如下所示：
+您可以查看 [`yolov5_s-static.py`](../../../configs/deploy/model/yolov5_s-static.py) 中测试流程或测试数据集加载部分，如下所示：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:25]:
```suggestion
您可以查看 [`yolov5_s-static.py`](https://github.com/open-mmlab/mmyolo/tree/main/configs/deploy/model/yolov5_s-static.py) 中测试流程或测试数据集加载部分，如下所示：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ test_dataloader = dict(
 
 #### (2) 部署配置文件
 
-当您部署在 `ONNXRuntime` 时，您可以查看 [`detection_onnxruntime_static.py`](configs/deploy/detection_onnxruntime_static.py) ，如下所示：
+当您部署在 `ONNXRuntime` 时，您可以查看 [`detection_onnxruntime_static.py`](../../../configs/deploy/detection_onnxruntime_static.py) ，如下所示：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:58]:
```suggestion
当您部署在 `ONNXRuntime` 时，您可以查看 [`detection_onnxruntime_static.py`](https://github.com/open-mmlab/mmyolo/tree/main/configs/deploy/detection_onnxruntime_static.py) ，如下所示：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 4]
```diff
@@ -75,9 +75,9 @@ codebase_config = dict(
 backend_config = dict(type='onnxruntime')
 ```
 
-默认配置中的 `post_processing` 后处理参数是当前模型与 `pytorch` 模型精度对齐的配置，若您需要修改相关参数，可以参考 [`部署必备指南`](./部署必备指南.md) 的详细介绍。
+默认配置中的 `post_processing` 后处理参数是当前模型与 `pytorch` 模型精度对齐的配置，若您需要修改相关参数，可以参考 [`部署必备指南`](basic_deployment_guide.md) 的详细介绍。
 
-当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_static-640x640.py`](config/deploy/detection_tensorrt_static-640x640.py) ，如下所示：
+当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_static-640x640.py`](../../../configs/deploy/detection_tensorrt_static-640x640.py) ，如下所示：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:80]:
```suggestion
当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_static-640x640.py`](https://github.com/open-mmlab/mmyolo/tree/main/configs/deploy/detection_tensorrt_static-640x640.py) ，如下所示：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 5]
```diff
@@ -102,7 +102,7 @@ use_efficientnms = False
 
 #### (1) 模型配置文件
 
-当您需要部署动态输入模型时，模型的输入可以为任意尺寸(`TensorRT` 会限制最小和最大输入尺寸)，因此使用默认的 [`yolov5_s-v61_syncbn_8xb16-300e_coco.py`](configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py) 模型配置文件即可，其中数据处理和数据集加载器部分如下所示：
+当您需要部署动态输入模型时，模型的输入可以为任意尺寸(`TensorRT` 会限制最小和最大输入尺寸)，因此使用默认的 [`yolov5_s-v61_syncbn_8xb16-300e_coco.py`](../../../configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py) 模型配置文件即可，其中数据处理和数据集加载器部分如下所示：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:105]:
```suggestion
当您需要部署动态输入模型时，模型的输入可以为任意尺寸(`TensorRT` 会限制最小和最大输入尺寸)，因此使用默认的 [`yolov5_s-v61_syncbn_8xb16-300e_coco.py`](https://github.com/open-mmlab/mmyolo/tree/main/configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py) 模型配置文件即可，其中数据处理和数据集加载器部分如下所示：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 6]
```diff
@@ -170,7 +170,7 @@ backend_config = dict(type='onnxruntime')
 
 与静态输入配置仅有 `_base_ = ['./base_dynamic.py']` 不同，动态输入会额外继承 `dynamic_axes` 属性。其他配置与静态输入配置相同。
 
-当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_dynamic-192x192-960x960.py`](config/deploy/detection_tensorrt_dynamic-192x192-960x960.py) ，如下所示：
+当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_dynamic-192x192-960x960.py`](../../../configs/deploy/detection_tensorrt_dynamic-192x192-960x960.py) ，如下所示：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:173]:
```suggestion
当您部署在 `TensorRT` 时，您可以查看 [`detection_tensorrt_dynamic-192x192-960x960.py`](https://github.com/open-mmlab/mmyolo/tree/main/configs/deploy/detection_tensorrt_dynamic-192x192-960x960.py) ，如下所示：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 7]
```diff
@@ -318,9 +318,9 @@ python3 ${MMDEPLOY_DIR}/tools/test.py \
 
 # 使用 Docker 部署测试
 
-`MMYOLO` 提供了一个 [`Dockerfile`](docker/Dockerfile_deployment) 用于构建镜像。请确保您的 `docker` 版本大于等于 `19.03`。
+`MMYOLO` 提供了一个 [`Dockerfile`](../../../docker/Dockerfile_deployment) 用于构建镜像。请确保您的 `docker` 版本大于等于 `19.03`。
 
-温馨提示；国内用户建议取消掉 [`Dockerfile`](docker/Dockerfile_deployment) 里面 `Optional` 后两行的注释，可以获得火箭一般的下载提速：
+温馨提示；国内用户建议取消掉 [`Dockerfile`](../../../docker/Dockerfile_deployment) 里面 `Optional` 后两行的注释，可以获得火箭一般的下载提速：
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:323]:
```suggestion
温馨提示；国内用户建议取消掉 [`Dockerfile`](https://github.com/open-mmlab/mmyolo/blob/main/docker/Dockerfile_deployment) 里面 `Optional` 后两行的注释，可以获得火箭一般的下载提速：
```

File: docs/zh_cn/deploy/yolov5_deployment.md [hunk 8]
```diff
@@ -318,9 +318,9 @@ python3 ${MMDEPLOY_DIR}/tools/test.py \
 
 # 使用 Docker 部署测试
 
-`MMYOLO` 提供了一个 [`Dockerfile`](docker/Dockerfile_deployment) 用于构建镜像。请确保您的 `docker` 版本大于等于 `19.03`。
+`MMYOLO` 提供了一个 [`Dockerfile`](../../../docker/Dockerfile_deployment) 用于构建镜像。请确保您的 `docker` 版本大于等于 `19.03`。
```
Comment (RangeKing) [inline docs/zh_cn/deploy/yolov5_deployment.md:321]:
```suggestion
`MMYOLO` 提供了一个 [`Dockerfile`](https://github.com/open-mmlab/mmyolo/blob/main/docker/Dockerfile_deployment) 用于构建镜像。请确保您的 `docker` 版本大于等于 `19.03`。
```

File: docs/en/deploy/basic_deployment_guide.md [hunk 9]
```diff
@@ -1 +1,287 @@
...
+- `max_output_boxes_per_class`: set the maximum number of bboxes for each class
+- `pre_top_k`: set the number of fixedcandidate bboxes before `nms`, sorted by scores
+- `keep_top_k`: set the number of output candidate bboxs after `nms`
+- `background_label_id`: set to `-1` as MMYOLO has no background class information
+
+### Configuration for Static Inputs
+
+#### 1. Model Config
+
+Taking the `YOLOv5` in MMYOLO as an example, here are the details:
```
Comment (PeterH0323) [inline docs/en/deploy/basic_deployment_guide.md:58]:
```suggestion
Taking `YOLOv5` of MMYOLO as an example, here are the details:
```

File: docs/en/deploy/basic_deployment_guide.md [hunk 10]
```diff
@@ -1 +1,287 @@
...
+
+`_base_ = '../../yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py'` inherits the model config in the training stage.
+
+`test_pipeline` adds the data processing piple for the deployment, `LetterResize` controls the size of the input images and the input for the converted model
+
+`test_dataloader` adds the dataloader config for the deployment, `batch_shapes_cfg` decides whether to use the `batch_shapes` strategy. More details can be found at [yolov5 configs](../user_guides/config.md)
+
+#### 2. Deployment Config
+
+Here we still use the `YOLOv5` in MMYOLO as the example. We can use \[detection_onnxruntime_static.py\`\](../../../configs/deploy/detection_onnxruntime_static.py) as the config to deploy \`YOLOv5\` to \`ONNXRuntim\` with static inputs.
```
Comment (PeterH0323) [inline docs/en/deploy/basic_deployment_guide.md:90]:
```suggestion
Here we still use the `YOLOv5` in MMYOLO as the example. We can use [`detection_onnxruntime_static.py`](../../../configs/deploy/detection_onnxruntime_static.py) as the config to deploy `YOLOv5` to `ONNXRuntime` with static inputs.
```

File: docs/en/deploy/basic_deployment_guide.md [hunk 11]
```diff
@@ -1 +1,287 @@
...
+        iou_threshold=0.5,
+        max_output_boxes_per_class=200,
+        pre_top_k=5000,
+        keep_top_k=100,
+        background_label_id=-1),
+    module=['mmyolo.deploy'])
+backend_config = dict(type='onnxruntime')
+```
+
+`backend_config` indicates the deployment backend with `type=‘onnxruntime’`, other information can be referred from the third section.
```
Comment (PeterH0323) [inline docs/en/deploy/basic_deployment_guide.md:110]:
```suggestion
`backend_config` indicates the deployment backend with `type='onnxruntime'`, other information can be referred from the third section.
```

File: docs/zh_cn/deploy/basic_deployment_guide.md [hunk 12]
```diff
@@ -135,7 +135,7 @@ use_efficientnms = False
 与 `ONNXRuntime` 部署配置不同的是，`TensorRT`  需要指定输入图片尺寸和构建引擎文件需要的参数，包括：
 
 - `onnx_config` 中指定 `input_shape=(640, 640)`
-- `backend_config['common_config']` 中 `fp16_mode=False` 和 `max_workspace_size=1 << 30`, `fp16_mode` 表示是否以 `fp16` 的参数格式构建引擎，`max_workspace_size` 表示当前 `gpu` 设备最大显存, 单位为 `GB`。`fp16` 的详细配置可以参考 [`detection_tensorrt-fp16_static-640x640.py`](configs/deploy/detection_tensorrt-fp16_static-640x640.py)
+- `backend_config['common_config']` 中包括 `fp16_mode=False` 和 `max_workspace_size=1 << 30`, `fp16_mode` 表示是否以 `fp16` 的参数格式构建引擎，`max_workspace_size` 表示当前 `gpu` 设备最大显存, 单位为 `GB`。`fp16` 的详细配置可以参考 [`detection_tensorrt-fp16_static-640x640.py`](configs/deploy/detection_tensorrt-fp16_static-640x640.py)
```
Comment (hhaAndroid) [inline docs/zh_cn/deploy/basic_deployment_guide.md:138]: configs/deploy/detect -> ../../../configs/deploy/detect

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**